src/components/checkout/payment-methods.js

```diff
@@ -0,0 +1,53 @@
+// Libraries
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+// Lib
+import componentMapping from '../../lib/component-mapping'
+
+class PaymentMethods extends Component {
+  constructor (props) {
+    super(props)
+    this.Button = componentMapping('Button')
+    this.PaymentMethodHeader = componentMapping('PaymentMethodHeader')
+    this.PayPalButton = componentMapping('PayPalButton')
+  }
+
+  /**
+   * When a customer selects the default checkout option, this funtion is called and 
+   * it sets the selected payment method in the state and redirects to the next section
+   */
+  handleDefaultPaymentSelection(paymentMethod) {
+    const { nextSection, handleSetPaymentMethod } = this.props
+    handleSetPaymentMethod(paymentMethod)
+    nextSection()
+  }
+
+  render () {
+    return (
+      <div aria-label='Payment method' className='o-form c-payment-methods'>
+        <this.PaymentMethodHeader title={'Payment Method'} />
+        <div className='c-payment-methods__options'>
+          <this.PayPalButton {...this.props}/>
```

Comment:
Can we pass the props down explicitly rather than with `...this.props`? The latter obfuscates the component's interface and makes debugging/refactoring harder in the future.